src/_P036_FrameOLED.ino

```diff
@@ -218,8 +278,12 @@ boolean Plugin_036(uint8_t function, struct EventStruct *event, String& string)
                           1,
                           P036_data_struct::getDisplaySizeSettings(tOLEDIndex).MaxLines);
       }
-      addFormNote(F("Will be automatically reduced if there is no font to fit this setting."));
-
+# ifdef P036_ENABLE_LEFT_ALIGN
+      {
+        addFormCheckBox(F("Reduce line number to fit font"), F("p036_ReduceLineNo"), bitRead(P036_FLAGS_1, P036_FLAG_REDUCE_LINE_NO));
+        addFormNote(F("When checked, 'Lines per Frame' will be automatically reduced to fit the individual line settings."));
```

Comment:
2 remarks here:
a) 'Reduce line number...' is wonky English, could be 'Reduce number of lines...' or 'Reduce nr. of lines...'

b) To reduce build size, the `addFormNote()` calls could be wrapped in `#ifndef LIMIT_BUILD_SIZE`, as the information shown is helpful but should *also* be reflected in the documentation. (This applies multiple times)

Reply:
Done!